controllers/sample_controller.py

```diff
@@ -35,3 +35,23 @@ def upload_sample(collection_exercise_id, file_path):
     logger.info('Successfully uploaded sample file',
                 collection_exercise_id=collection_exercise_id, sample_file=file_path)
     return response_json
+
+
+def upload_social_sample(collection_exercise_id, file_path):
```

Comment:
This function is very similar to the upload_sample function. Please merge the two and parameterise the function instead.